stable/rabbitmq/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.networkPolicy.enabled }}
+kind: NetworkPolicy
+apiVersion: {{ template "rabbitmq.networkPolicy.apiVersion" . }}
```

Comment:
Where is defined this `rabbitmq.networkPolicy.apiVersion`>

Reply:
Indeed, fixed by hardcoding it (since it has been out since kube 1.7)

Reply:
Indeed, fixed by hardcoding it (since it has been out since kube 1.7)